spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/mongo/MongoPropertiesTests.java

```diff
@@ -55,48 +58,43 @@ void canBindCharArrayPassword() {
 	@Test
 	@SuppressWarnings("deprecation")
 	void allMongoClientOptionsCanBeSet() {
```

Comment:
By restoring the code as it was before in my polish, this test fails. The replicat set is erased by the connection URI. I don't know if that's a new behaviour of the driver or if I messed up the change but I'd appreciate some help there.